web/client/utils/openlayers/DrawSupportUtils.js

```diff
@@ -12,12 +12,15 @@ import { reproject } from '../CoordinatesUtils';
 import { getCenter } from 'ol/extent';
 import { Circle } from 'ol/geom';
 
-const calculateRadius = (center, coordinates) => {
-    return isArray(coordinates) && isArray(coordinates[0]) && isArray(coordinates[0][0]) ? Math.sqrt(Math.pow(center[0] - coordinates[0][0][0], 2) + Math.pow(center[1] - coordinates[0][0][1], 2)) : 100;
+const calculateRadius = (center, coordinates, mapCrs, featureCrs) => {
+    if (isArray(coordinates) && isArray(coordinates[0]) && isArray(coordinates[0][0])) {
+        const point = reproject(coordinates[0][0], featureCrs, mapCrs);
+        return Math.sqrt(Math.pow(center[0] - point.x, 2) + Math.pow(center[1] - point.y, 2));
+    }
+    return 100;
 };
 
-export const transformPolygonToCircle = (feature, mapCrs) => {
-
+export const transformPolygonToCircle = (feature, mapCrs, featureCrs = mapCrs) => {
```

Comment:
I know that you just changed the pre-existing code, but can you add some unit tests for the transformPolygonToCircle function as well of some documentation?